recipe/meta.yaml

```diff
@@ -18,18 +18,43 @@ requirements:
     - python
     - pip
   run:
-    - python
+    - entrypoints
+    - ipython
+    - jsonschema
+    - numpy
     - pandas
-    - vega 0.4.4
```

Comment:
Should this version depend on `vega3`?
If so we need to wait for https://github.com/conda-forge/staged-recipes/pull/5731

Reply:
There's been some discussion of dependencies here: https://github.com/altair-viz/altair/issues/751

My feeling at this point is that we should probably not depend on IPython, jupyter, notebook, or vega, because there are ways to use altair without them.

Reply:
That said, IPython is still a runtime dependency in rc2, but it's now a soft dependency in master.

Reply:
So this is OK for `rc2` then, right? Should we merge it?

The goal of this PR is to provide users the same install options via `pip` or `conda`, even though conda support for RCs kind of sucks at the moment... The pip version,

```
pip install --pre altair
```

is much better than the conda version

```
conda install --channel conda-forge/label/dev altair
```